.github/workflows/docker.yml

```diff
@@ -3,62 +3,27 @@ name: Docker CI
 on:
   push:
     branches: [ master ]
-    tags:
-      - "*.*.*"
+    paths:
+      - Dockerfile
+      - tools/install-dependencies
```

Comment:
Can you remove paths? Let's always run on master

Reply:
Yes, done